.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Conversion to new code style
+3626843dcac55d6f264a61251e745406159ea2e8
```

Comment:
Shouldn't this be exactly the commit published in #6120? It looks like 3626843dcac55d6f264a61251e745406159ea2e8 won't be in git history of master branch.

Reply:
After the merger, 6020 will need to be checked and corrected.